src/components/ui/chart.tsx

```diff
@@ -181,7 +187,25 @@ const ChartTooltipContent = React.forwardRef<
           className
         )}
       >
-        {!nestLabel ? tooltipLabel : null}
+        {!nestLabel && tooltipLabel ? (
+          <div className="flex">
+            {tooltipLabel}
+            {row.holidayName && row.holidayName !== "" ? (
+              <span className="text-red-500 ml-2">{row.holidayName}</span>
+            ) : row.dayOfWeek && row.dayOfWeek !== "" ? (
```

Comment:
```suggestion
              <span className="text-red-500 ml-2">{row.holidayName} </span>
            ) : undefined }
            {row.dayOfWeek && row.dayOfWeek !== "" ? (
```

マウスオーバー時のポップアップには曜日も書いてあったほうが便利に思います 🤔